inference-engine/thirdparty/clDNN/src/graph_optimizer/compile_graph.cpp

```diff
@@ -10,17 +10,64 @@
 #include "program_node.h"
 #include "cldnn/runtime/engine.hpp"
 #include "runtime/cldnn_itt.hpp"
+#include <iostream>
+#include <cmath>
+#include <iomanip>
+
+#define CLDNN_THREADING_SEQ 0
+#define CLDNN_THREADING_TBB 1
```

Comment:
these defs exist in both kernels_cache.cpp and here. could we move this to more common header? e.g., engine.hpp?

Reply:
Okay, I see. I will change it. thanks

Reply:
move them to engine.hpp. Thanks.